finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -396,6 +410,7 @@ mod tests {
         let expected_missing: Vec<_> = (1..NUM_NODES).map(NodeIndex).collect();
         assert_eq!(missing_nodes, expected_missing);
         assert!(handler0.peer_id(&NodeIndex(1)).is_none());
+        assert!(handler0.authentication_for(&NodeIndex(1)).is_none());
     }
 
     #[tokio::test]
```

Comment:
I guess it is worth checking whether `authentication_for` adapts well to the cached update

Reply:
Added check.